R/wilcoxonsr_hodges_lehman.qmd

```diff
@@ -0,0 +1,118 @@
+
+### Examples {.unnumbered}
+
+```{r}
+# Exact 
+stats::wilcox.test(x = blood_p$bp_after, y = blood_p$bp_before, 
+                                   paired = TRUE, 
+                                   conf.int = TRUE, 
+                                   conf.level = 0.9, 
+                                   alterative = "two.sided", 
+                                   exact = TRUE)
+
+# No exact & continuity correction
+stats::wilcox.test(x = blood_p$bp_after, y = blood_p$bp_before, 
+                                   paired = TRUE, 
+                                   conf.int = TRUE, 
+                                   conf.level = 0.9, 
+                                   alterative = "two.sided", 
+                                   exact = FALSE, 
+                                   correct = TRUE)
+
+
+# No exact & No continuity correction
+stats::wilcox.test(x = blood_p$bp_after, y = blood_p$bp_before, 
+                                     paired = TRUE, 
+                                     conf.int = TRUE, 
+                                     conf.level = 0.9, 
+                                     alterative = "two.sided" , 
+                                     exact = FALSE, 
+                                     correct = FALSE)
+
+```
+
+### Important notes on stats:wilcox.test {.unnumbered}
+
+-   By default an exact p-value is computed if the samples contain less than 50 finite values and there are no ties. Otherwise, a normal approximation is used.
+-   If exact p-values are available, an exact confidence interval is obtained by the algorithm described in Bauer (1972), and the Hodges-Lehmann estimator is employed. Otherwise, the returned confidence interval and point estimate are based on normal approximations.
+-   If non-exact p-value is calculated, continuity correction in the normal approximation for the p-value can be applied with *correct* argument.
+-   Statistic V is provided, which is a test statistic based on Bauer algorithm
+
+#### DOS2 {.unnumbered}
+
+Function *senWilcox* used for Sensitivity Analysis for Wilcoxon's Signed-rank Statistic will be applied. For more information about that function go [here](https://www.rdocumentation.org/packages/DOS/versions/1.0.0/topics/senWilcox)
+
+### Examples {.unnumbered}
+
+```{r}
+DOS2::senWilcox(blood_p$bp_after - blood_p$bp_before, 
+                   gamma = 1, 
+                   conf.int = TRUE, 
+                   alpha = 0.1, 
+                   alternative = "twosided")
+```
+
+### Important notes on DOS2:senWilcox {.unnumbered}
+
+-   Gamma \>= 1 is the value of the sensitivity parameter. If gamma=1, then you are assuming ignorable treatment assignment or equivalently no unmeasured confounding - that is the considered scenario in our example, sensitivity analysis is not performed.
+-   Only p value, estimate and CI are provided
+
+### Important notes on coin package {.unnumbered}
```

Comment:
Did this get cut off? Or did you mean to delete the coin package? You have commented out the lib call up above, but did you mean to delete it?